src/app/app.config.ts

```diff
@@ -42,6 +42,7 @@ function initializeAppFactory(userGrpcService: UserGrpcService, userService: Use
           throw new Error('No user');
         }
         userService.user = data.user;
+        navigationService.refreshSidebar();
```

Comment:
I don't think you have to do this, since it already initialize the sidebar on its construction (when the service is provided for the first time)